src/erf.cpp

```diff
@@ -58,11 +59,15 @@ int main(int argc, char **argv) {
 		uint32 id = kERFID;
 		Common::UString archive;
 		Aurora::ERFWriter::Version version = Aurora::ERFWriter::kERFVersion10;
+		Aurora::ERFWriter::Compression compression = Aurora::ERFWriter::kCompressionNone;
 		std::set<Common::UString> files;
 
-		if (!parseCommandLine(args, returnValue, archive, files, version, id, game))
+		if (!parseCommandLine(args, returnValue, archive, files, version, compression, id, game))
```

Comment:
You're not passing that compression value to Aurora::ERFWriter in line 74, though